### ISSUE TITLE
The 'accessibilityLabel' prop is not accepting by the component. Automation framework is unable to find test ID's.

### ISSUE THREAD

**Issue by vetrivendhan48** [magicismight/react-native-root-toast#151]
Snippet
`Toast.show(_localize(message), {
            // opacity: 1,
            duration: Toast.durations.LONG,
            backgroundColor: isError ? '#CC0000' : '#007E33',
            position: Toast.positions.TOP,
            shadow: true,
            animation: true,
            hideOnPress: true,
            delay: 0,
            textStyle: { flex: 1, fontSize: 13 },
            onShow: () => {
                // isCommonToastActive = true
            },
            onShown: () => {
                // calls on toast\`s appear animation end.
            },
            onHide: () => {
                // calls on toast\`s hide animation start.
            },
            onHidden: () => {
                // isCommonToastActive = false
            },
            testID: `toast message`,
            accessibilityLabel: `toast message`
        });`

The prop **accessibilityLabel** is not displayed in automation test tools(Testproject).
Is there is any other way to achieve this? Thanks.

**Comment by seba9999** [comment]
+1 woul love to have accessibilityLabel ... :'(
